.github/workflows/check-pr.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - name: "Run configure"
         run: |
-          ./ccex configure
+          ./ccex configure format
```

Comment:
As default, `./ccex configure` prepares format and test environments.

This job needs format environment only.

Reply:
LGTM. I'll approve this PR after CI is fixed.